lumigator/python/mzai/backend/backend/api/routes/experiments.py

```diff
@@ -1,8 +1,65 @@
-from fastapi import APIRouter
+from uuid import UUID
+
+from fastapi import APIRouter, status
+from lumigator_schemas.experiments import (
+    ExperimentCreate,
+    ExperimentResponse,
+    ExperimentResultDownloadResponse,
+    ExperimentResultResponse,
+)
+from lumigator_schemas.extras import ListingResponse
+from lumigator_schemas.jobs import (
+    JobEvalCreate,
+)
+
+from backend.api.deps import JobServiceDep
 
 router = APIRouter()
 
 
+@router.post("/", status_code=status.HTTP_201_CREATED)
+def create_experiment(
+    service: JobServiceDep,
+    request: ExperimentCreate,
+) -> ExperimentResponse:
+    return service.create_job(JobEvalCreate.model_validate(request.model_dump()))
+
+
+@router.get("/{experiment_id}")
+def get_experiment(service: JobServiceDep, experiment_id: UUID) -> ExperimentResponse:
+    return ExperimentResponse.model_validate(
+        service.get_job(experiment_id).model_dump()
+        )
+
+
 @router.get("/")
-def get_experiments():
-    return {"message": "Experiments"}
+def list_experiments(
+    service: JobServiceDep,
+    skip: int = 0,
+    limit: int = 100,
+) -> ListingResponse[ExperimentResponse]:
+    return ListingResponse[ExperimentResponse].model_validate(
+        service.list_jobs(skip, limit).model_dump()
```

Comment:
Right, this has to be the short-term compromise. The drawback is any job (even if it's just an inference job) will show up as an experiment.

Not for this PR, as this is something the Jobs service doesn't support either, but it feels like we should be able to return jobs of a specific type (e.g. evaluation only) --> #380